### ISSUE TITLE
Issue with no-mixed-spaces-and-tabs rule

### ISSUE THREAD

**Issue by gyandeeps** [eslint/eslint#1294]
For this scenario:

``` .js
var a = 1; \nfunction minus(x, y){\n    return x - y;\n}\nfunction add(x, y){\n\treturn x + y;\n}
```

It doesn't flag the above scenario as warning/error.
According to me it should.
Is this an issue?
